### DISCUSSION TITLE
Manually triggered image updates

### DISCUSSION THREAD

**Discussion by jakubhajek** [fluxcd/flux2#1572] [Q&A]
Hello Flux contributors!

Thanks a lot for creating such a great tool!

I would like to use the Image Automation feature. However, I have a specific requirement that updating the image should be triggered manually by the operator instead of automatically updated immediately while the new image is available at the repo.

Is this possible to configure that kind of behaviour?

Thank you,

**Comment by stefanprodan** [comment]
Yes you can run `flux suspend/resume image repository/automation`. You can either toggle suspension for specific `ImageRepositories` or for all images in a namespace by toggling the `ImageUpdateAutomation` object.

**Reply by jakubhajek** [reply]
Hi Stefan, 

thanks a lot for your answer. Actually, I was thinking of using suspend/resume in order to manually trigger image updates.
